README.md

```diff
@@ -163,10 +170,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="$OVN_CENTRAL_IP:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
```

Comment:
@salv-orlando 
No problem, I am still learning python packaging via pip. So don't assume I know my stuff here. I am just figuring things out by reading docs and experimenting. So please feel free to say I am wrong!

When I was using 'pip install -e', I was noticing what you explained. But now when I do 'pip install .', the executables are directly installed (without the re-direction). May be there is a subtlety here?